ros_ws/src/car_control/launch/car_control.launch

```diff
@@ -20,6 +20,8 @@ Launches all necessary nodes that listen for drive_parameters and translate them
       type="dms_controller"
       name="dms_controller"
       output="screen" >
+      <param name="dms_check_rate" type="int" value = "20" /> <!-- in Hz -->
+      <param name="dms_expiration" type="int" value = "100" /> <!-- in ms -->
```

Comment:
Maybe add <arg> tags so these parameters can be changed from the command line?

Reply:
> Do you have an example?

Look into src/teleoperation/launch/remote_control.launch